doc/manual.rst

```diff
@@ -4219,8 +4221,8 @@ Raise statement
 
 Example:
 
-.. code-block:: nim
-  raise newEOS("operating system failed")
+.. code-block:: nim/
```

Comment:
Why the forward slash?

Reply:
Remove the slash please.

Reply:
Sorry that was a typing error. I'll change it.

Reply:
> The CIs do check the documentation so don't use the `[ci skip]` for it please.
Okay.